src/Orc.NuGetExplorer.Xaml/Orc.NuGetExplorer.Xaml.Shared/ViewModels/PackageDetailsViewModel.cs

```diff
@@ -42,7 +53,31 @@ public PackageDetailsViewModel(IPackageDetails package, IPackageDetailsService p
         {
             await base.InitializeAsync();
 
-            PackageSummary = _packageDetailsService.PackageToFlowDocument(Package);
+            if (Package is ModelBase modelBase)
+            {
+                modelBase.PropertyChanged += (sender, args) =>
+                    {
+                        if (args.PropertyName == nameof(Package.SelectedVersion))
```

Comment:
if (args.HasPropertyChanged(nameof(Package.SelectedVersion))

Because string.Empty should cause this to run as well